README.md

```diff
@@ -44,7 +44,9 @@ The optional parameters are:
 20. ***DISABLE_OPS_AGENT***. Can be one of:
     - `false` (default): Install Ops Agent with random-backoff retries
     - `true`: Do not install Ops Agent
-
+21. ***DISABLE_NOTEBOOK***. Can be one of:
```

Comment:
It is usually better practice to use ENABLE as config name vs DISABLE

Reply:
If changed, we probably need to change DISABLE_OPS_AGENT too.

Reply:
changed.